app/workers/test_user_dashboard/daily_maintenance.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module TestUserDashboard
+  class DailyMaintenance
+    include Sidekiq::Worker
+
+    TUD_ACCOUNTS_TABLE = 'tud_accounts'
+
+    def perform
+      checkin_tud_accounts
+      mirror_tud_accounts_in_bigquery
+    end
+
+    private
+
+    def checkin_tud_accounts
+      TestUserDashboard::TudAccount.where.not(checkout_time: nil).each do |account|
+        # rubocop:disable Rails/SkipsModelValidations
```

Comment:
I think you may want to see the Rubocop docs [here](https://www.rubydoc.info/gems/rubocop/0.51.0/RuboCop/Cop/Rails/SkipsModelValidations) for `Rails/SkipsModelValidations`. I believe you might be able to follow the #good example

Reply:
Thanks Lindsey. I make an 'update' to that. Har har har :-)